website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -352,10 +365,6 @@ NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
 memcached-operator-controller-manager   1/1     1            1           8m
 ```
 
-### 3. Deploy your Operator with the Operator Lifecycle Manager (OLM)
-
-OLM will manage creation of most if not all resources required to run your operator, using a bit of setup from other `operator-sdk` commands. Check out the [docs][cli-run-olm] for more information.
-
```

Comment:
we cannot add this step here since we did not describe the steps to integrate the project with OLM in the tutorial. However, instead of that, we add as the next step a suggestion to check how to integrate the project with it.

Reply:
That's why the OLM integration docs are linked. Please revert this change.